db/db.sql

```diff
@@ -14,8 +14,8 @@ CREATE TABLE `jobs` (
   `request` varchar(4096) not null default 'unknown',
   `isAsync` boolean,
   `numInputGranules` integer not null default 0,
-  `collectionIds` text not null
-);
+  `collectionIds` text not null,
+  `shapeFileHash` varchar(255));
```

Comment:
Not sure if we want to set it to `varchar(32)` since an md5 checksum should be 32 characters and if not set it will be an empty string.